pinot-controller/src/main/resources/app/pages/TenantDetails.tsx

```diff
@@ -558,6 +572,13 @@ const TenantPageDetails = ({ match }: RouteComponentProps<Props>) => {
               >
                 Rebalance Brokers
               </CustomButton>
+              <CustomButton
+                onClick={handleRepairTable}
+                tooltipTitle="Repair table"
+                enableTooltip={true}
+              >
+                Repair Table
+              </CustomButton>
```

Comment:
the title says repair realtime table. do we need to show this option only for realtime tables?

Reply:
yes. fixed